src/standalone/Selector/MultiSelectWithTags.tsx

```diff
@@ -74,6 +101,26 @@ export interface MultiSelectWithTagsProps<Data extends MultiSelectData>
 	 * Label for search input control
 	 */
 	searchInputLabel?: string;
+	/**
+	 * Display switch control
+	 */
+	displaySwitch?: boolean;
```

Comment:
make switchLabel, switchValue and handleSwitch required if displaySwitch is true
```tsx
interface MultiSelectWithTagsSwitchPropsOn {
  displaySwitch: true;
  switchLabel: string;
  switchValue: boolean;
  handleSwitch: (...) => ;
}
interface MultiSelectWithTagsSwitchPropsOff {
  displaySwitch?: false;
}
type MultiSelectWithTagsSwitchProps = MultiSelectWithTagsSwitchPropsOn | MultiSelectWithTagsSwitchPropsOff;
interface Props extends MultiSelectWithTagsSwitchProps {
...
}
```